docs/package/gluon-scheduled-domain-switch.rst

```diff
@@ -31,8 +33,11 @@ Example::
     target_domain = 'new_domain',
     switch_after_offline_mins = 120,
     switch_time = 1546344000, -- 01.01.2019 - 12:00 UTC
-    connection_check_targets = {
+  },
+
+  check_connection = {
```

Comment:
I'd prefer to call this `connection_check`, not `check_connection`. (Also in the package name)

Reply:
the idea was to add more of those packages later: check_gw_connection, check_xyz,...

Reply:
ok, so we keep the current name check_connection